sphinx/templates/latex/longtable.tex_t

```diff
@@ -1,3 +1,11 @@
+<%- if table.align -%>
```

Comment:
OMG, I'd forgotten to review the rendering result of longtable.
It doesn't work. I will update this later.